python/feather/ext.pyx

```diff
@@ -95,6 +95,11 @@ cdef class FeatherWriter:
             PrimitiveArray levels
 
         col_values = _unbox_series(col)
+        if (col_values.codes == -1).any():
+            if mask is None:
+                mask = col_values.codes == -1
+            else:
+                mask = mask | col_values.codes == -1
```

Comment:
Can you avoid computing the `col_values.codes == -1` more than once (save it in a temporary variable)?